roles/kube_nginx_ingress/tasks/main.yml

```diff
@@ -1,16 +1,29 @@
+- name: 'create tempfile for nginx-ingress values'
+  tempfile:
+    state: file
+    suffix: yml
+  register: nginx_ingress_value_file
+
+- debug:
+    var: nginx_ingress_value_file.path
+  when: debug|bool
+
+- name: 'copy nginx-ingress values into temporary file'
+  copy:
+    src: nginx_ingress_values.yml
+    dest: '{{ nginx_ingress_value_file.path }}'
+
 - name: 'install nginx ingress'
   command: >-
-    helm install stable/nginx-ingress
-    --name nginx-ingress
+    helm upgrade --install nginx-ingress stable/nginx-ingress
+    --version 0.11.1
     --namespace kube-ingress
-    --set controller.hostNetwork=true
-    --set controller.kind=DaemonSet
-    --set rbac.create=true
+    -f {{ nginx_ingress_value_file.path }}
   run_once: True
   delegate_to: "{{ groups['kube-master'][0] }}"
-  register: helm_nginx_ingress
-  failed_when: >-
-    helm_nginx_ingress.rc != 0 and
-    'already exists' not in helm_nginx_ingress.stderr|default('')
-  changed_when: >-
-    'already exists' not in helm_nginx_ingress.stderr|default('')
+
+- name: 'remove uneeded nginx-ingress values file'
```

Comment:
`unneeded`

Reply:
I would like to find a proper path on the system to allow futur admin to edit and test modification directly (without ansible), but now it's a temporary file in /tmp/, it's commonly admitted that you need to cleanup your file when you finished with it ?

Reply:
No no, I meant, there's a typo :smiley: